packages/client-common/src/config.ts

```diff
@@ -397,6 +438,9 @@ export function loadConfigOptionsFromURL(
             }
             config.keep_alive.enabled = booleanConfigURLValue({ key, value })
             break
+          case 'access_token':
+            config.auth = { access_token: value }
```

Comment:
Is it OK to leave it as just `access_token` (similarly to what we currently have with `username` and `password`, so it is somewhat consistent), or make a key that resembles the full path (`auth_access_token`)?

Reply:
The URL config example needs to be updated.

Reply:
what is the benefit of the full path `auth_access_token`?